rep-0147.rst

```diff
@@ -59,7 +59,7 @@ These messages definitions were used to help define this set.
  - mavros_msgs [#mavros]_
  - nav_msgs [#nav_msgs]_
  - trajectory_msgs [#trajectory_msgs]_
-
+ - as2_msgs [#as2_msgs]_
```

Comment:
I don't really think this is appropriate.  As far as I can tell, the `as2_msgs` were not used to define this set.

Reply:
Removed from this PR.